hasher-matcher-actioner/webapp/src/components/HolidaysDatasetInformationBlock.jsx

```diff
@@ -21,7 +21,7 @@ export function HolidaysDatasetInformationBlock({samplePGExists, refresh}) {
       `This group will become disabled on "Sync" as it does not have TE
       counterpart. Simply delete and create again if you wish to keep match
       sample data after syncing`,
-    ).then(() => {});
```

Comment:
A eslint requested change. Tested this by deleting and recreating the holiday dataset in the UI

Reply:
hmm, I think not waiting on the future at all is a slight change in logic but the only case I could maybe see that mattering is if they click off the page immediately. Probably fine